spikeinterface/widgets/timeseries.py

```diff
@@ -37,10 +36,15 @@ class TimeseriesWidget(BaseWidget):
         If True groups are plotted with different colors
     color:   str default: None
         The color used to draw the traces.
-    clim: None or tupple
-        When mode='map' this control color lims
+    clim: None, tuple, or list of tuples
+        When mode='map' this control color lims. 
+        If list of tuples, each entry is the color lims for one layer.
     with_colorbar: bool default True
         When mode='map' add colorbar
+    tile_size: int
+        For figurl backend, the size of each tile in the redered image
```

Comment:
This should not be int but something in second or millisecond and then internally converted to tile_size

Reply:
oups no sorry I mixed witth seconds_per_row